docs/subscriptions/index.md

```diff
@@ -58,6 +58,14 @@ Datetime deleted
 }
 :::
 
+:::{admonition} Tombstoning
+:class: note
+
+As is implicit from the description of the servers-side behavior for the different calls, servers SHOULD hold all previous `guid` and `feed_url` field data with a link to the succeeding data (such that a path of values can be followed) or with a link to the most recent data. This in order to handle situations where clients submit old data. For example:
```

Comment:
I think it could be good to point out that only the old GUID and the link to the new entry is necessary to store and that the rest can be deleted.

We only describe this behaviour here:
>If the GUID isn’t already present, create a new subscription entry and update the existing entry’s new_guid field to point to the newly created entry

It could also be enough to just add a sentence to this section like: `The fields GUID and NEW_GUID of the existing entry SHOULD be kept indefinitely for referencing.`